applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -167,6 +167,7 @@ static void print_codec(const struct bt_audio_codec_cfg *codec, enum bt_audio_di
 	if (codec->id == BT_HCI_CODING_FORMAT_LC3) {
 		/* LC3 uses the generic LTV format - other codecs might do as well */
 		enum bt_audio_location chan_allocation;
+		int dur_us = le_audio_duration_us(codec);
```

Comment:
When these functions are called, we need to perform a check: `dur_us <= 0`
If so, return an error.

If `le_audio_duration_us` returns an error here, we will print the negative number.

Reply:
If I assert in the two get functions for an error with the ID or actual Hz/us value this would simplify the code and trap this error. Would this be acceptable?

Reply:
Reccomend we do the usual err_check as an Assert will reboot the device, and it quite harsh.

Reply:
We let the layer above decide which action to take.